place_tracker/lib/place_map.dart

```diff
@@ -4,6 +4,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
 import 'package:uuid/uuid.dart';
+import 'package:provider/provider.dart';
```

Comment:
nit: This should be sorted with the rest of the list.

Reply:
should I add a new line before this import statement?

Reply:
@tusharojha No. You need to sort the imports alphabetically. You can take a look at how imports are arranged in other dart files.

Reply:
got it

Reply:
@jddeep can you check it, I think i have already done this.